src/client/js/Panels/MetaEditor/MetaEditorControl.DiagramDesignerWidgetEventHandlers.js

```diff
@@ -479,6 +492,10 @@ define(['js/logger',
                 self._deletePointerRelationship(connDesc.GMESrcId, connDesc.GMEDstId, connDesc.name, false);
             } else if (connDesc.type === MetaRelations.META_RELATIONS.INHERITANCE) {
                 self._deleteInheritanceRelationship(connDesc.GMESrcId, connDesc.GMEDstId);
+            } else if (connDesc.type === MetaRelations.META_RELATIONS.MIXIN) {
```

Comment:
Why do we have the exact same condition twice? Exact same function call, this is suspicious.